libnmstate/schemas/operational-state.yaml

```diff
@@ -457,3 +467,14 @@ definitions:
         type: array
         items:
           type: string
+  vlan-filter:
+    type: object
+    properties:
+      vlan-range-min:
+        type: integer
+        minimum: 0
```

Comment:
I think this is 1 as stated at my previous comment

Reply:
Done.

Reply:
Fixed in https://github.com/nmstate/nmstate/pull/440/